f2/src/utils/regex.js

```diff
@@ -2,5 +2,6 @@ export const regexDef = () => {
   return {
     phoneRegExp: /^(\+?\d{0,4})?\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{4}\)?)?$/,
     postalCodeRegex: /^[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJKLMNPRSTVWXYZ]( )?\d[ABCEGHJKLMNPRSTVWXYZ]\d$/i,
+    isValidInternationalPhonenumber: /^\d{7,}$/,
```

Comment:
Is this simply checking for a 7 digit (or greater) number?

Reply:
This is what valid international number is required.

Reply:
If we are validating international phone numbers shouldn't we use E.164 format instead of just check if it is 7 or more digits?